less/home-page.less

```diff
@@ -1,31 +1,51 @@
 .home {
+  header {
+    // display: flex;
+    .flexboxall(column, no-wrap, center);
+    border-bottom: 2px dashed @navigation-border;
+
+  }
 
   .content-section {
+    max-width: 900px;
     margin: 30px 0;
     display: flex;
     flex-wrap: wrap;
     justify-content: space-between;
 
-   
-    
+
     .text-content {
       width: 48%;
       padding-right: 1%;
-    }
+      text-align: left;
 
```

Comment:
instead of text align, try achieving the same thing using a flex component such as `justify-content: flex-start`